### DISCUSSION TITLE
Solved : Library won't see converted .mp3 files

### DISCUSSION THREAD

**Discussion by phreatic62** [rern/rAudio#596] [Fixed Issues]
Update: Solved. ID10T problem, not a bug. :)

As it says on the tin really.
I have some .wma files which the library won't 'see', so I used VLC and converted them to .mp3 and it's still won't add them to the library.

Which sort of begs another question, should rAudio recognise .wma and .ogg files?

Edit, the converted files play in Groo9ve Music, so they are not totally corrupted.

**Comment by rern** [comment]
Settings > Player > Options > FFmpeg - enable

( **For filetype:** 16sv 3g2 3gp 4xm 8svx aa3 aac ac3 adx afc aif aifc aiff al alaw amr anim apc ape asf atrac au aud avi avm2 avs bap bfi c93 cak cin cmv cpk daud dct divx dts dv dvd dxa eac3 film flac flc fli fll flx flv g726 gsm gxf iss m1v m2v m2t m2ts m4a m4b m4v mad mj2 mjpeg mjpg mka mkv mlp mm mmf mov mp+ mp1 mp2 mp3 mp4 mpc mpeg mpg mpga mpp mpu mve mvi mxf nc nsv nut nuv oga ogm ogv ogx oma ogg omg opus psp pva qcp qt r3d ra ram rl2 rm rmvb roq rpl rvc shn smk snd sol son spx str swf tak tgi tgq tgv thp ts tsp tta xa xvid uv uv2 vb vid vob voc vp6 vmd wav webm wma wmv wsaud wsvga wv wve rtp:// rtsp:// rtsps:// )

**Reply by phreatic62** [reply]
Thanks for the reply, I tried that and still no joy. It's (sort of) working, I can browse to the files in the library if I select my NAS and drill down, I can even set to ignore a folder and un-ignore it, however when I update the library, it doesn't add it in the list of albums or artists.
Update: I deleted and re added one album, then ddi an update of the library (only changes) and it added it, but this did not work for another album that I can see in the library on the NAS, but not with albums or artists.

**Comment by kan-ibal** [comment]
You are facing two separate issues. First with ability to play and codecs answered above and second with id3 tags for album and artist recognition. Check and edit tags for that files using tag editor, kid3 in linux or mp3tag for win.

**Reply by phreatic62** [reply]
Yes, thank you, this appears to be it, I've tried mp3tag on one album (flac) and it now shows up!
